provisioner/ibm-s3fs-provisioner.go

```diff
@@ -637,6 +645,22 @@ func (p *IBMS3fsProvisioner) Provision(ctx context.Context, options controller.P
 			}
 		}
 
+		if pvc.BucketVersioning != "" {
+			enable := pvc.BucketVersioning == "true"
```

Comment:
`pvc.BucketVersioning` need to be converted to lowercase before comparision and spaces need to be trimmed.